test/extended/util/client.go

```diff
@@ -423,6 +423,23 @@ func (c *CLI) SetupWorkSpaceWithSpecificPath(serverURL string) {
 	e2e.Logf("Workspace %q has been fully provisioned.", c.currentWs.Name)
 }
 
+// ListWorkSpaces returns a list of WorkSpaces under the home workspace
+func (c *CLI) ListWorkSpaces() []string {
+	return c.ListWorkSpacesWithSpecificPath(c.homeServerURL)
+}
+
+// ListWorkSpacesWithSpecificPath returns a list of WorkSpaces under a specific workspace
+func (c *CLI) ListWorkSpacesWithSpecificPath(serverURL string) []string {
+	workSpacesRaw, err := c.WithoutNamespace().WithoutKubeconf().Run("get").Args("workspaces", "-o", `jsonpath="{.items[*]['metadata.name']}"`, "--server="+serverURL).Output()
+	o.Expect(err).NotTo(o.HaveOccurred())
+	workSpacesRaw = strings.Trim(workSpacesRaw, " ")
+	workSpacesRaw = strings.Trim(workSpacesRaw, `"`)
```

Comment:
@zimo-xiao line 435, 436, we can use `strings.Trim(workSpacesRaw, ` "`)` to do that.